BUILD

```diff
@@ -12,6 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@rules_license//rules:license.bzl", "license")
```

Comment:
Should the commit heading read "Add `@rules_license`-style license declarations" instead?

Reply:
Yes. Done.